source/uwp/AdaptiveCardsNuget/AdaptiveCards.UWP.Beta.native.targets

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <AdaptiveCards-Platform Condition="'$(Platform)' == 'Win32'">x86</AdaptiveCards-Platform>
```

Comment:
during the meeting, i understood as this is ok with you.

---
In reply to: [134582740](https://github.com/Microsoft/AdaptiveCards/pull/568#discussion_r134582740) [](ancestors = 134582740)